spec/spec_helper.rb

```diff
@@ -9,7 +9,16 @@
 require "pry"
 require "lita/rspec"
 
+# Disable deprecation warnings for Redis.
+# Remove this line and address each warning before using v5+ of the redis gem as a dependency.
+Redis.silence_deprecations = true
+
 RSpec.configure do |config|
+  unless ENV.key?("CI")
+    config.filter_run focus: true
+    config.run_all_when_everything_filtered = true
+  end
```

Comment:
Helpful (for me anyways) when debugging spec failures, allows you to easily run a single spec or group of specs with `focus: true` attribute.